htdocs/fourn/facture/contact.php

```diff
@@ -215,30 +214,40 @@
 		}
 		print '</td></tr>';
 
+
 		// Label
-		print '<tr><td>'.$form->editfieldkey("Label", 'label', $object->label, $object, 0).'</td><td>';
-		print $form->editfieldval("Label", 'label', $object->label, $object, 0);
+		print '<tr><td>'.$langs->trans('Label').'</td>';
```

Comment:
Why removing the use of ->editfieldkey
In a future all fields in form may use this so we will be able to make "inline edition".

Reply:
on a undedicaced tabs i find that it's not a good way to have fields updatable
replace all the display fields by inline edition on dolibarr is an very huge work for small usage by the way
why not continue to keep it simple?